docs/conf.py

```diff
@@ -26,11 +26,16 @@
 
 # Add any Sphinx extension module names here, as strings. They can be extensions
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
-extensions = ['sphinx.ext.autodoc', 'sphinx.ext.intersphinx',
-              #'sphinx.ext.autosummary', 'sphinx.ext.pngmath',
+extensions = ['sphinx.ext.autodoc',
+              #'sphinx.ext.autosummary',
               'numpydoc',
               'imageio_ext', ]
 
+# Monkey-patch numpydoc to don't do the autosummary thing
+import docscrape_sphinx
+assert docscrape_sphinx.SphinxDocString._str_member_list
+docscrape_sphinx.SphinxDocString._str_member_list = lambda self, name: []
```

Comment:
I can confirm that the fix I propose results in an [error-free build](http://debomatic-amd64.debian.net/distribution#unstable/python-imageio/2.1.0+ds1-1/buildlog). The end of the tunnel is near @almarklein :-)